Tests/FluentAssertions.Specs/Primitives/ReferenceTypeAssertionsSpecs.cs

```diff
@@ -42,8 +42,8 @@ public void When_two_different_objects_are_expected_to_be_the_same_it_should_fai
         act
             .Should().Throw<XunitException>()
             .WithMessage(
-                "Expected subject to refer to*{ UserName = JohnDoe } because " +
-                "they are the same, but found*{ Name = John Doe }.");
+                @"Expected subject to refer to*{*UserName = ""JohnDoe""*} because " +
```

Comment:
When introducing `space` after `{` and before `}` in case not creating "new line" the change on this test should get smaller.

Reply:
This test changed because the `UserName`/`Name` properties are indented on new lines now:

    Expected subject to refer to 
    {
        UserName = "JohnDoe"
    } because they are the same, but found 
    {
        Name = "John Doe"
    }.